CHANGELOG.md

```diff
@@ -642,6 +642,15 @@ Anyway, the cluster should get back to the proper size after 10 min.
          the new CRI + CNI code path.
       * If you are using plugins other than `bridge`, make sure you have
         updated custom plugins to the latest version that is compatible.
+* **CNI plugins now affect node readiness**
+  * Kubelet will now block node readiness until a CNI configuration file is
+    present in /etc/cni/net.d or a given custom CNI configuration path.
+    [This change](https://github.com/kubernetes/kubernetes/pull/43474) ensures
+    kubelet does not start pods that require networking before
+    [networking is ready](https://github.com/kubernetes/kubernetes/issues/43014).
+    This change may require changes to clients if the client gates pod scheduling
```

Comment:
@jbeda I don't think that statement is accurate.  Previously, *all* pods would be scheduled, some (hostNetwork:true) would run successfully, and the rest would fail over and over and over in a loop forever because networking wasn't ready.

Kubelet still allows hostNetwork:true pods to be run on the node even if the node isn't ready (AFAIK) so "This is no longer possible" isn't correct.

It's simply that clients that assume node readiness means *no* pods can be scheduled can no longer assume that.